server/src/currencyList.py

```diff
@@ -0,0 +1,53 @@
+# Dependencies:
+# Need to run python3 -m pip install requests 
+import requests
+import json
+
+# Setup URL string
+
+# EFFECT: Returns the list of currencies in the form of an array
+def currency_list():
+    API_KEY = "b248938740450b2e6e0616db"
+
+    URL = "https://v6.exchangerate-api.com/v6/" + API_KEY + "/latest/CAD"
```

Comment:
It might be better to put the base URL in global scope (i.e. `BASE_URL = "https://v6.exchangerate-api.com/v6/" + API_KEY`